### PR TITLE
uth: use gfortran

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: src/arch/uth-linux-x86_64/conv-mach.sh [hunk 1]
```diff
@@ -1,8 +1,7 @@
 . $CHARMINC/cc-gcc.sh
+. $CHARMINC/conv-mach-gfortran.sh
```
Comment (evan-charmworks) [inline src/arch/uth-linux-x86_64/conv-mach.sh]: This should probably be:
```
if command -v gfortran >/dev/null 2>&1
then
  . $CHARMINC/conv-mach-gfortran.sh
fi
```
See #1925

File: src/arch/uth-linux/conv-mach.sh [hunk 2]
```diff
@@ -1,8 +1,7 @@
 . $CHARMINC/cc-gcc.sh
+. $CHARMINC/conv-mach-gfortran.sh
```
Comment (evan-charmworks) [inline src/arch/uth-linux/conv-mach.sh]: Same

File: src/arch/uth-linux-x86_64/conv-mach.sh (hunk 1 above)
Comment (matthiasdiener) [reply src/arch/uth-linux-x86_64/conv-mach.sh]: done.

Comment (matthiasdiener) [reply src/arch/uth-linux-x86_64/conv-mach.sh]: done

**Comment by matthiasdiener** [comment]
Needs another review.

**Pull request merged**